lib/logMessages.js

```diff
@@ -0,0 +1,52 @@
+const template_Install_Started_Msg ='Template installation started because you passed --install flag';
```

Comment:
this const should not be created, it is as you can see used only in tests, instead, in tests you should use the log function that you created. Something like:
```js
installationDebugMessage(logMessages.TEMPLATE_INSTALL_FLAG_MSG);
```

Reply:
Umm okay , I will just simply return it from that method.